index.html

```diff
@@ -0,0 +1,311 @@
+        </div>
+      </div>
+      <div class="row">
+        <div class="banner_box">
+          <b>Open collaborative writing with Manubot</b><br />
+          Daniel S. Himmelstein, Vincent Rubinetti, David R. Slochower, Dongbo Hu, Venkat S. Malladi, Casey S. Greene, Anthony Gitter<br />
+          <i>Manubot Preprint</i> (2019)
+          <a href="https://greenelab.github.io/meta-review/">
+            https://greenelab.github.io/meta-review/
+          </a>
+        </div>
+      </div>
+    </section>
+    <section>
+      <div class="section_header">
+        Acknowledgements
+      </div>
+      <div class="row">
+        <b>Funded by</b>
+      </div>
+      <div class="row">
+        <figure>
+          <a href="https://sloan.org/"><img  src="alfred-p-sloan-foundation.png" height="80px" /></a>
+          <figcaption>
+            <a href="https://sloan.org/grant-detail/8501">Grant G-2018-11163</a>
+          </figcaption>
+        </figure>
+        <figure>
+          <a href="https://www.moore.org/"><img  src="gordon-betty-moore-foundation.png" height="80px" /></a>
+          <figcaption>
+            <a href="https://www.moore.org/grant-detail?grantId=GBMF4552">Grant GBMF4552</a>
+          </figcaption>
+        </figure>
+      </div>
+      <div class="row">
+        <b>Led by</b>
+      </div>
+      <div class="row">
+        <figure>
+          <a href="https://github.com/dhimmel"><img class="acknowledgement_image" src="daniel-himmelstein.jpg" height="150px" /></a>
+          <figcaption>
+            <span class="semibold">Daniel Himmelstein</span><br />
+            <a href="http://www.greenelab.com/">Greene Lab</a><br />
+            <a href="https://www.upenn.edu/">Univ. of Pennsylvania</a>
+          </figcaption>
+        </figure>
+        <figure>
+          <a href="https://github.com/agitter"><img class="acknowledgement_image" src="anthony-gitter.jpg" height="150px" /></a>
+          <figcaption>
+            <span class="semibold">Anthony Gitter</span><br />
+            <a href="https://morgridge.org/research/virology/gitter-lab/">Morgridge Institute</a><br />
+            <a href="https://www.biostat.wisc.edu/~gitter/">Univ. of Wisconsin-Madison</a>
+          </figcaption>
+        </figure>
+      </div>
+      <div class="row">
+        <div>
+          As an open source project, Manubot relies on a community of maintainers, contributors, and users.
+          For a list of all the individuals who have made Manubot possible, see <a href="https://github.com/manubot/rootstock/graphs/contributors">the contributors on GitHub</a> and <a href="https://greenelab.github.io/meta-review/">the authors and acknowledgements here</a>.
+        </div>
+      </div>
+    </section>
+    <footer>
+      <a href="https://github.com/manubot">
+        <i class="fab fa-lg fa-github"></i>
+      </a>
+      <a href="https://twitter.com/dhimmel">
```

Comment:
Should we replace the twitter URL with https://twitter.com/search?q=%23Manubot until we get a handle?

Reply:
The Twitter search makes sense to me as a placeholder.